flink-kubernetes-operator/src/main/java/org/apache/flink/kubernetes/operator/config/FlinkConfigManager.java

```diff
@@ -87,8 +89,15 @@ public Configuration load(Key k) {
                                 });
 
         updateDefaultConfig(defaultConfig);
+        ScheduledExecutorService executorService = Executors.newSingleThreadScheduledExecutor();
```

Comment:
It seems that the `executorService` is never shutdown. It might not be a major problem because the `FlinkConfigManager` has the same lifecycle with whole process.

Reply:
I thought about this but the operator doesn't really offer lifecycle proper lifecycle methods.